core/domain/stats_jobs_one_off.py

```diff
@@ -117,16 +117,30 @@ def map(item):
                         feconf.EVENT_TYPE_STATE_HIT, item.created_on,
                         item.exploration_id))
                 return
+            encoded_state_name = item.state_name
```

Comment:
So is the idea here to make this state name into a unicode string?

If so, might be clearer to say:

`unicode_state_name = item.state_name if isinstance(item.state_name, unicode) else item.state_name.decode('utf-8')`

Reply:
Done.